cursorless-talon/src/apps/cursorless_vscode.py

```diff
@@ -4,6 +4,8 @@
 
 ctx.matches = r"""
 app: vscode
+# Disable Cursorless for vscode file dialogs
+not title: /^(Open Folder|Open File|Save As|Open Workspace from file|Add Folder to Workspace|Save Workspace)$/i
```

Comment:
```suggestion
not win.title: /^(Open Folder|Open File|Save As|Open Workspace from File|Add Folder to Workspace|Save Workspace)$/i
```

Reply:
I think either work fine and we should resolve the inconsistency in community as a followup, slight reference for this personally